src/app/aether-device-group/device-group-edit/device-group-edit.component.ts

```diff
@@ -81,7 +82,7 @@ export class DeviceGroupEditComponent
                 Validators.maxLength(80),
             ]),
         ],
-        'ip-domain': [undefined],
+        'ip-domain': [{ value: '', disabled: true }],
```

Comment:
the purpose of `undefined` was to not set an empty stting value, so it think the equvalent would be:
```typescript
'ip-domain': [{ value: undefined, disabled: true }]
```

Reply:
Done